tools/BUILD.bazel

```diff
@@ -99,3 +100,26 @@ container_image(
     user = "root",
     visibility = ["//visibility:public"],
 )
+
+# bash tar layer for docker images.
+genrule(
+    name = "bash_tar",
```

Comment:
why is bash_tar and libtinfo6_tar needed, does one handle something differently than the other?
perhaps related to comment # bash tar layer for docker images.

Reply:
Added commentary to explain this